build/.vsts-ci.yml

```diff
@@ -23,11 +23,11 @@ jobs:
     displayName: 'build'
 - job: windowsx86
   pool:
-    name: 'aziotbld-win03'
+    vmImage: 'vs2017-win2016'
   steps:
   - script: |
      if exist jenkins\windows_c.cmd (
-      call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\Common7\Tools\VsDevCmd.bat"
+      call "%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsDevCmd.bat"
```

Comment:
pleading ignorance here why the `%` when line 358 doesn't have it

Reply:
fixed